app/templates/_package.json

```diff
@@ -1,30 +1,48 @@
 {
-  "name": "<%= appname %>",
-  "version": "0.0.0",
+  "name": "<%= generatorName %>",
+  "version": "0.0.1",
   "description": "Yeoman generator",
-  "license": "MIT",
-  "main": "app/index.js",
-  "repository": "<%= githubUser %>/<%= appname %>",
+  "keywords": [
+    "javascript",
+    "yeoman-generator",
+    "generator",
+    "inception",
+    "init",
+    "create",
+    "boilerplate",
+    "template",
+    "scaffold"
+  ],
   "author": {
-    "name": "<%= realname %>",
-    "email": "<%= email %>",
-    "url": "<%= githubUrl %>"
+    "name": "<%= gitUser.name || gitUser.login %>",
+    "email": "<%= gitUser.email %>",
+    "url": "<%= gitUser.htmlUrl %>"
   },
+  "main": "app/index.js",
+  "repository": "<%= gitUser.login %>/<%= generatorName %>",
   "scripts": {
-    "test": "mocha"
+    "test": "./node_modules/.bin/istanbul cover ./node_modules/mocha/bin/_mocha -- test/node"
```

Comment:
npm is adding the locally install binaries to the path when running script.

So these are not using the globally installed version, they're using the local ones ;)